package.json

```diff
@@ -0,0 +1,783 @@
+    "token-types": "^4.1.1",
+    "tough-cookie": "^4.0.0",
+    "tr46": "^2.1.0",
+    "trim-newlines": "^3.0.1",
+    "trough": "^1.0.5",
+    "tsconfig-paths": "^3.11.0",
+    "tslib": "^2.3.1",
+    "tsutils": "^3.21.0",
+    "tunnel": "^0.0.6",
+    "tunnel-agent": "^0.6.0",
+    "tweetnacl": "^0.14.5",
+    "type-check": "^0.4.0",
+    "type-fest": "^0.20.2",
+    "type-is": "^1.6.18",
+    "typedarray-to-buffer": "^3.1.5",
+    "typescript": "^4.4.3",
+    "unbox-primitive": "^1.0.1",
+    "unbzip2-stream": "^1.3.3",
+    "unified": "^9.2.2",
+    "unique-filename": "^1.1.1",
+    "unique-slug": "^2.0.2",
+    "unique-string": "^2.0.0",
+    "unist-util-is": "^4.1.0",
+    "unist-util-find-all-after": "^3.0.2",
+    "universalify": "^2.0.0",
+    "unpipe": "^1.0.0",
+    "unist-util-stringify-position": "^2.0.3",
+    "uri-js": "^4.4.1",
+    "update-notifier": "^5.1.0",
+    "url": "^0.11.0",
+    "url-parse-lax": "^3.0.0",
+    "util-deprecate": "^1.0.2",
+    "utila": "^0.4.0",
+    "utils-merge": "^1.0.1",
+    "uuid": "^3.4.0",
+    "v8-compile-cache": "^2.3.0",
+    "validate-npm-package-license": "^3.0.4",
+    "validate-npm-package-name": "^3.0.0",
+    "vary": "^1.1.2",
+    "verror": "^1.10.0",
+    "vfile": "^4.2.1",
+    "vfile-message": "^2.0.4",
+    "w3c-xmlserializer": "^2.0.0",
+    "w3c-hr-time": "^1.0.2",
+    "watchpack": "^2.2.0",
+    "wbuf": "^1.7.3",
+    "wcwidth": "^1.0.1",
+    "webidl-conversions": "^6.1.0",
+    "webpack-dev-middleware": "^5.2.1",
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.1",
+    "websocket-extensions": "^0.1.4",
+    "websocket-driver": "^0.7.4",
+    "whatwg-encoding": "^1.0.5",
+    "whatwg-url": "^8.7.0",
+    "whatwg-mimetype": "^2.3.0",
+    "wide-align": "^1.1.3",
+    "widest-line": "^3.1.0",
+    "which-boxed-primitive": "^1.0.2",
+    "which": "^2.0.2",
+    "word-wrap": "^1.2.3",
+    "wildcard": "^2.0.0",
+    "wrap-ansi": "^7.0.0",
+    "wrappy": "^1.0.2",
+    "ws": "^8.2.3",
+    "write-file-atomic": "^3.0.3",
+    "xml-name-validator": "^3.0.0",
+    "xdg-basedir": "^4.0.0",
+    "xmlbuilder": "^11.0.1",
+    "xml2js": "^0.4.23",
+    "xmlchars": "^2.2.0",
+    "yallist": "^4.0.0",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "yocto-queue": "^0.1.0",
+    "yauzl": "^2.10.0",
+    "zwitch": "^1.0.5"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.3.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.24.2",
+    "hint": "^6.1.4",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.3.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.56.1",
+    "webpack-cli": "^4.8.0",
+    "webpack-dev-server": "^4.3.1"
+  },
+  "scripts": {
+    "test": "jest",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
```

Comment:
- [ ] Everything looks great, but I'm afraid I couldn't install the project locally...
- I clone the repo and select the structure branch
- I delete the `package-lock.json`
- I ran `npm i` and I have the next error:
![image](https://user-images.githubusercontent.com/20563252/136140073-1414168d-5824-470b-9e59-40f609a0ca10.png)
- For this same reason I couldn't use `npm start` (and this is a requirement for this project), please fix this bug. 🐶